### ISSUE TITLE
API may no longer be working

### ISSUE THREAD

**Issue by JlNX** [Akianonymus/SMLoader#1]
Attempted to login using current credentials and the app responded wrong credentials. Confirmed I was using the correct credentials.

Version: 1.9.4 (64bit)
OS: Windows 10 Pro (Build 17134.590) (64-Bit)

![image](https://user-images.githubusercontent.com/13512575/53892650-4e4fd100-3ff2-11e9-9c67-2192b4917256.png)

**Comment by lockntross** [comment]
Same, it's a sad moment. A small monument will be erected to remember SMLoader. RIP.

**Comment by realReXishere** [comment]
Yeah I've trying to download a playlist but the credential error won't let me pass.

RIP

**Comment by realReXishere** [comment]
Just researched a bit. The updated API is working.
Download it from here
https://git.fuwafuwa.moe/SMLoadrDev/SMLoadr

**Comment by tinnted** [comment]
Same issue for me and my friend as well
